### DISCUSSION TITLE
Custom Domain Pricing

### DISCUSSION THREAD

**Discussion by switchupcb** [zitadel/zitadel#7049] [Q&A] [answered]
### Describe your problem

A dedicated server with 2 vCPU 4 GB RAM is $40/m.

Centralized Zitadel costs $100 per month after 100 daily active users (with "unlimited users and organizations").

So, a person must choose between self-hosting for <$40/m on custom domain (e.g., `sso.mydomain.com`) + a few hours of setup **OR** use Centritadel, which costs over $50/m for a custom domain. 

I'm not exactly sure what the difference between self-hosted vs. Centritadel is, other than the fact that there is no automatic monitoring of https://github.com/zitadel/zitadel/security issues on a self-hosted infrastructure. In addition to the topics covered in this blog post: https://zitadel.com/blog/saas-vs-self-hosted

So, I am wondering whether there is something I am overlooking. Perhaps, I am not the target audience for those tiers.

When a person has the technical capacity to self-host Zitadel on Kubernetes, why would they use this $50/m option and 100-user limit tier. Does the $50/m pricing for a custom domain make sense or am I just a stupid poor?

### Describe your ideal solution

Custom Domain Pricing for $12/m

Or create a tier for ameripoors that costs one minimum wagie.

**Comment by doncicuto** [comment]
Hi @hifabienne and @switchupcb IMHO maybe this issue should be moved to GitHub discussions and/or Discord chat as this this is not a technical improvement issue that can be tackled with a PR.

**Comment by hifabienne** [comment, marked as answer]
Hei @switchupcb 
Choosing our managed ZITADEL Cloud service has the advantages that you do not have to care about the infrastructure, updates, backups, monitoring, database management, etc. We also give you an guarantee on the availability, and some support.
I understand that the infrastructure costs are probably lower if you are running ZITADEL yourself. But the infrastructure costs are only a part of it. Often the time required for setting up, running, maintaining, etc is often forgotten.

In regard of the custom domains, as soon as you upgrade to the pro tier you also have one custom domain included.
